### DISCUSSION TITLE
工单执行能否增加回调机制

### DISCUSSION THREAD

**Discussion by dc-vin** [hhyo/Archery#1830] [Ideas]
目前工单执行完成，会有钉钉通知，不过我们基于openapi做了开发，非常期望能够回调结果，这样方便我们做统计

**Comment by dc-vin** [comment]
还有一个小问题，我看有其他用户讨论新的发行版本1.9.0，但是在github上貌似没有

**Reply by LeoQuote** [reply]
这个暂未发布呢， 你用 master 分支就可以了

**Reply by dc-vin** [reply]
公司项目不好随意改，我还是等一波release，让运维去升级

**Comment by LeoQuote** [comment]
我之前也提过一个类似的想法： https://github.com/hhyo/Archery/discussions/1580 ， 暂时没时间做， 欢迎 pr

**Reply by dc-vin** [reply]
现阶段我对python熟练度不高，暂时帮不上忙😂
